src/modeling_library/distributions/binom.jl

```diff
@@ -0,0 +1,29 @@
+struct Binomial <: Distribution{Int} end
+
+"""
+	binom(n::Integer, p::Real)
+Sample an `Int` from the Binomial distribution with parameters `n` (number of
+trials) and `p` (probability of success in each trial).
+"""
+const binom = Binomial()
+
+function logpdf(::Binomial, x::Integer, n::Integer, p::Real)
+	if x < 0 return -Inf end
+	coefficient = loggamma(n + 1) - loggamma(n + 1 - x) - loggamma(x + 1)
+	return coefficient + x * log(p) + (n - x) * log(1 - p)
```

Comment:
Can this instead call the implementation in Distributions.jl?

Reply:
@bzinberg is this just for consistency w the `neg_binom` implementation? Because it doesn't seem like we have a standard way to do this now -- some of the built-in distributions call out to Distributions.jl for logpdf, and some don't.

Reply:
It's not for consistency, it's because re-implementing things creates more surface for bugs (including future changes to one of the copies that forget to touch the other one).  So I think we should use the existing implementations in Distribitions.jl (given that we already depend on it) where possible, unless there is particular reason not to.

Reply:
I agree that we should always use existing code in Distributions.jl whenever possible.

The implementations of `logpdf` that do not use Distributions.jl are a relic from an earlier time when gradients of logpdf were implemented by applying ReverseDiff.jl to `logpdf` (so it had to be written in a special way so that ReverseDiff.jl could work with it, and the implementations in Distribution.jl were not written that way). Now, we have `logpdf_grad` implemented separately, so all new distributions have `logpdf` that just call Distributions.jl when available.

Reply:
Cool that makes sense -- just made the change! :)